app/components/requests_feed/requests_feed.rb

```diff
@@ -14,7 +14,7 @@ def initialize(organization:, contributor:, **)
     attr_reader :organization, :contributor
 
     def replies
-      @replies ||= contributor.recent_replies
+      @replies ||= contributor.most_recent_replies_to_some_request
```

Comment:
```suggestion
      @replies ||= contributor.recent_replies_to_a_request
```

Reply:
I updated it to `most_recent_reply_to_each_request` to accurately describe what it returns.
I also moved the method to the request feed class since it is the only place it is used.